sdk/cosmos/azure-cosmos-test/src/main/java/com/azure/cosmos/test/util/Beta.java

```diff
@@ -0,0 +1,49 @@
+package com.azure.cosmos.test.util;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import static java.lang.annotation.ElementType.CONSTRUCTOR;
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.TYPE;
+
+/**
+ * Indicates functionality that is in preview and as such is subject to change in non-backwards compatible ways in future releases,
+ * including removal, regardless of any compatibility expectations set by the containing library version.
+ */
+@Documented
+@Retention(RetentionPolicy.CLASS)
+@Target({ TYPE, METHOD, PARAMETER, CONSTRUCTOR, FIELD })
+@Inherited
+public @interface Beta {
```

Comment:
No Beta-tag needed here - we would just annotate the entire package azure-cosmos-test.jar as beta (when releasing to Maven) until (and if at all) we want to GA it in Maven